Example/InstanceID/Tests/FIRInstanceIDKeyPairMigrationTest.m

```diff
@@ -154,31 +154,15 @@ - (void)testUpdateKeyRefWithTagRetainsAndReleasesKeyRef {
 
     publicKeyRef = keyPair.publicKey;
 
-    // Retain to keep publicKeyRef alive to verify its reatin count
-    CFRetain(publicKeyRef);
-
-    // 2 = 1 from keyPair + 1 from CFRetain()
-    XCTAssertEqual(CFGetRetainCount(publicKeyRef), 2);
-
-    XCTestExpectation *completionExpectaion =
-        [self expectationWithDescription:@"completionExpectaion"];
+    XCTestExpectation *completionExpectation =
+        [self expectationWithDescription:@"completionExpectation"];
     [self.keyPairStore updateKeyRef:keyPair.publicKey
                             withTag:@"test"
                             handler:^(NSError *error) {
-                              [completionExpectaion fulfill];
+                              [completionExpectation fulfill];
                             }];
-
-    // 3 = from keyPair + 1 from CFRetain() + 1 retained by `updateKeyRef`
-    XCTAssertEqual(CFGetRetainCount(publicKeyRef), 3);
   }
-
-  // 2 = 1 from CFRetain() + 1 retained by `updateKeyRef`
-  XCTAssertEqual(CFGetRetainCount(publicKeyRef), 2);
```

Comment:
I think this implementation option will work if we check that `publicKeyRef` can be accessed here. E.g.
```
XCTAssertNoThrow(publicKeyRef);
```

Otherwise this test does not really tests what is declared...